vital/util/file_md5.cxx

```diff
@@ -64,7 +65,9 @@ file_md5( const string& fn )
     while ( is )
     {
       is.read( reinterpret_cast< char* >( buf ), bufsize );
-      if ( is.gcount() )
+      //gcount() should never be negative, but fortify does not seem to
```

Comment:
It's ok to keep this here, but kwiversys::MD5.c:md5_append:383 does the check and returns `if (nbytes <= 0)`

Edit* Noticed the diff and that we always had the if stmt here. This is fine.